lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
+const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   create() {
     this.serverless.cli.log('Creating Stack...');
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+    let stackTags = { STAGE: this.options.stage };
```

Comment:
@flomotlik sure. Please bear in mind that users might have AWS Config rules that require all CloudFormation stacks to be explicitly tagged (good example is 'maintainer-email'). These rules would reject creation of the stack if the tag is undefined.